### ISSUE TITLE
Script error while using Commanche self designate

### ISSUE THREAD

**Issue by bux578** [acemod/ACE3#3219]
ACE3 Version: 3.4.2 stable

**Mods:**
* `@CBA_A3`
* `@ace`

**Placed ACE3 Modules:**
* None

**Description:**
```
9:58:32 Error in expression <lect 2)))];
};
};
};
};
};


if (_laser params [["_laserPos", [], [[]], 3], ["_l>
 9:58:32   Error position: <params [["_laserPos", [], [[]], 3], ["_l>
 9:58:32   Error Params: Type Array, expected Number
 9:58:32 File z\ace\addons\laser\functions\fnc_seekerFindLaserSpot.sqf, line 54
```

**Steps to reproduce:**
* Create dedicated server that has 2 slots
* Open two A3 instances and with both join the server
* Let player2 take a Laser Designator and let him mark a certain spot with a laser
* Let player1 spawn an empty Commanche pointing towards the laser target
* Let player1 hop into the Commanche as gunner and try to lock onto the laser target (default <kbd>R</kbd> (doesn't work, see #3211))
* Using the interact key, start the self designating process
* Fire a DAGR rocket -> script error

**Where did the issue occur?**
local "Multiplayer"

**RPT log file:**
http://pastebin.com/ZbsZnsZm

**Comment by bux578** [comment]
Related #3202

**Comment by bux578** [comment]
https://youtu.be/i6W_8woKvcM

**Comment by GitHawk** [comment]
![2016-02-10_00001](https://cloud.githubusercontent.com/assets/1816059/12954853/a15d0ccc-d020-11e5-9e9d-6b09dd289b6f.jpg)
Also happens with Ka-60 Kasatka

**Comment by GitHawk** [comment]
Should no longer happen in current build as #3305 disabled ace_laser for those missiles.
Will finally be fixed in #3308
